osidb/tests/endpoints/flaws/test_unembargo.py

```diff
@@ -176,6 +176,12 @@ def test_complex(self, auth_client, test_api_uri):
                 )
             )
 
+            assert (
+                flaw["acls_read"] == settings.PUBLIC_READ_GROUPS
+                and flaw["acls_write"] == [settings.PUBLIC_WRITE_GROUP]
+                for flaw in Flaw.objects.all()
+            )
```

Comment:
Ah okay, I will just extend the previous assertion to check read and write groups then.

Reply:
I was not testing the correct models, and I have fixed now this. though I didnt add the check for the write groups to the original tests since it's outside the scope of this feature.